src/AzureExtension/DevBox/Helpers/WingetConfigWrapper.cs

```diff
@@ -328,21 +323,60 @@ private void SetStateForCustomizationTask(TaskJSONToCSClasses.BaseClass response
         }
     }
 
+    // Start the Dev Box if it isn't already running
+    private async Task HandleNonRunningState()
+    {
+        // Check if the dev box might have been started in the meantime
+        if (_devBox.GetState() == ComputeSystemState.Running)
+        {
+            return;
+        }
+
+        // Start the Dev Box
+        try
+        {
+            _log.Information("Starting the dev box to apply configuration");
+            var startResult = await _devBox.StartAsync(string.Empty);
+        }
+        catch (Exception ex)
+        {
+            _log.Error(ex, "Unable to start the dev box");
+            throw new InvalidOperationException(Resources.GetResource(DevBoxErrorStateKey, _devBox.DisplayName));

```

Comment:
FYI: I think you meant to use `DevBox_ErrorStartKey` instead of the state key right?